website/docs/contributing/ADRs/back-end/specificity-db-columns.md

```diff
@@ -0,0 +1,75 @@
+
+**Note**: Aliases (`u` for `users`, `o` for `orders`) are used here for brevity and readability, but they are optional. The key aspect is specifying the table for each column.
+
+**2. Discouraged syntax: Implicit table naming**
+
+```ts
+const rows = await this.db
+    .select(
+        'id', 
+        'name', 
+        'email',
+        'description',
+    )
+    .from('users')
+    .join('orders', 'orders.user_id', 'users.id')
+    .where('status', 'active')
+    .orderBy('created_at', 'desc')
+```
+
+**Why discouraged**: Without specifying the table for each column, it becomes unclear which table each column belongs to. This ambiguity, especially in tables with identically named columns, can lead to runtime errors that are difficult to anticipate, particularly in queries involving joins.
+
+### Advantages
+
+The primary benefits of this approach are:
+
+1. **Clarity and reduced ambiguity**: By explicitly specifying table names for each column, we eliminate ambiguity about which table a column belongs to. This is particularly beneficial in complex queries involving multiple tables and joins. This also reduces the risk of ambiguity errors during database migrations or schema changes.
+
+2. **Ease of maintenance and adaptability to changes**: During database migrations or schema changes, specifically referenced columns make it easier to identify and update relevant queries. This reduces the risk of overlooking changes that might affect query behavior.
+
+3. **Improved readability in complex queries**: In queries involving multiple tables and joins, explicitly specifying table names makes the query more readable and understandable, facilitating easier debugging and review, while ensuring consistency across the codebase.
+
+### Concerns
+
+The adoption of this practice comes with minimal concerns:
+
+1. **Slight increase in verbosity**: The queries will be slightly longer due to the addition of table names in columns. However, the benefits in clarity and maintainability far outweigh this minor increase in verbosity.
+
+2. **Initial adaptation curve**: There might be a brief period of adjustment as developers adapt to this new standard. However, given its straightforward nature, this learning curve is expected to be minimal.
+
+## Conclusion
+
+The adoption of explicit table name specification in our SQL queries is a strategic decision aimed at improving the clarity, maintainability, and reliability of our database interactions. This practice will ensure that our queries remain robust and clear, especially in the context of evolving database schemas and complex query scenarios. We will gradually implement this standard in our existing codebase by following the Girl Scout Rule. This change aligns with our commitment to writing clean, understandable, and maintainable code, thereby enhancing the overall quality of our software development processes.
```

Comment:
I assume Girl Scout Rule is a modernization of the Boy scout rule concept to be more gender positive

Reply:
That was my suggestions. I think the language in general is very biased towards masculine forms and would like to shift the balance.